src/main/java/lotto/model/domain/Lotto.java

```diff
@@ -0,0 +1,72 @@
+package lotto.model.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public final class Lotto {
+
+    public static final int SIZE = 6;
+    public static final int COST = 1_000;
+
+    private final List<LottoNumber> lottoNumbers;
+
+    private Lotto(final List<Integer> numbers) {
+        validateSize(numbers);
+        validateDistinction(numbers);
+
+        this.lottoNumbers = numbers.stream()
+                .map(LottoNumber::of)
+                .collect(Collectors.toUnmodifiableList());
+    }
+
+    public static Lotto create(final List<Integer> numbers) {
+        return new Lotto(numbers);
+    }
+
+    private void validateSize(final List<Integer> numbers) {
+        if (numbers.size() != SIZE) {
+            throw new IllegalArgumentException("로또는 6개의 번호를 가져야합니다.");
```

Comment:
에러메시지는 어떤 상황에서 발생했는지 맥락을 부여해주면 훨씬 좋은 메시지가 됩니다. 
```suggestion
            throw new IllegalArgumentException("로또는 6개의 번호를 가져야합니다. 입력된 사이즈의 수 : " + lottos.size());
```